containers/docker-compose.yaml

```diff
@@ -52,7 +57,7 @@ services:
       - GDS_ADMIN_REVIEW_URL=http://localhost:3001/vasps/
       - GDS_CERTMAN_INTERVAL=5m
       - GDS_CERTMAN_STORAGE=/data/certs
-      - GDS_BACKUP_ENABLED=true
+      - GDS_BACKUP_ENABLED=false
```

Comment:
when configured with trtl, this needs to be false otherwise GDS will exit FATAL (can't backup)